src/azure-cli/azure/cli/command_modules/role/_help.py

```diff
@@ -397,21 +402,51 @@
 
 helps['ad group'] = """
 type: group
-short-summary: Manage Azure Active Directory groups.
+short-summary: Manage groups.
 """
 
 helps['ad group create'] = """
 type: command
-short-summary: Create a group in the directory.
+short-summary: Create a group.
```

Comment:
It is pretty obvious the group is created in the directory. If we mention `in the director` here, it should also be mentioned for `app,` `sp` and `user`.